ansible-ipi-install/roles/node-prep/templates/install-config.j2

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+baseDomain: {{ domain }}
+metadata:
+  name: {{ cluster }}
+networking:
+  machineCIDR: {{ extcidrnet }}
+compute:
+- name: worker
+  replicas: {{ numworkers }}
+controlPlane:
+  name: master
+  replicas: {{ nummasters }}
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: {{ apivip }}
+    ingressVIP: {{ ingressvip }}
+    dnsVIP: {{ ns1vip }}
```

Comment:
should be dnsvip

Reply:
does this even need to be specified? might be better to just resolve ns1, *.apps, etc. if something can be discovered, doing it that way can lead to less errors. you could also do an assert earlier on to ensure all the entries were created before hand, and bail otherwise

Reply:
@dcritch to my knowledge it does. However, I'll defer to others whom may know more than I. I'll chat with you Monday to discuss further.